backend/app/api/prices.py

```
@@ -20,18 +21,167 @@
+
+    logger.info(f"Complete history update finished: {total_added} added, {total_updated} updated")
+
+
+@router.get("/history/{symbol}")
+async def get_price_history(
+    symbol: str,
+    days: Optional[int] = None,
+    start_date: Optional[str] = None,
+    end_date: Optional[str] = None,
+    limit: Optional[int] = None,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.14.0)

Unused function argument: `db`

(ARG001)